src/Kentico.Xperience.Lucene.Core/InfoModels/LuceneReusableContentTypeItem/LuceneReusableContentTypeItemInfoProvider.generated.cs

```diff
@@ -0,0 +1,18 @@
+using CMS.DataEngine;
+
+namespace Kentico.Xperience.Lucene.Core;
+
+/// <summary>
+/// Class providing <see cref="LuceneReusableContentTypeItemInfo"/> management.
+/// </summary>
+[ProviderInterface(typeof(ILuceneReusableContentTypeItemInfoProvider))]
+public partial class LuceneReusableContentTypeItemInfoProvider : AbstractInfoProvider<LuceneReusableContentTypeItemInfo, LuceneReusableContentTypeItemInfoProvider>, ILuceneReusableContentTypeItemInfoProvider
```

Comment:
There is not need for dedicated InfoProvider. Just refactor Info class with IInfoProvider approach and remove this LuceneReusableContentTypeItemInfoProvider.